src/main/resources/localization/economy.json

```diff
@@ -10,18 +10,21 @@
     "Your recent novel becomes a hit, earning you {amount} in royalties.",
     "You find a rare fossil on the beach and sell it to the museum for {amount}.",
     "You invested in the right stocks a the right time, earning {amount}.",
-    "Your NFT collection surges in price, so you sell them for {amount}."
+    "Your NFT collection surges in price, so you sell them for {amount}.",
+    "You finished digging in Patrick's mines for the day, He pay's you {amount}."
   ],
   "crimeSuccess": [
     "You work for the Juarez cartel as a brick-presser and earn {amount}! They'll probably kill you and your family tomorrow. Just saying.",
     "You rob an orphanage for {amount}.",
     "You break into the house next door and sell their gaming PC on E-Bay for {amount}.",
-    "You work as a scam seller on Amazon selling fake airpods for full price and make {amount}."
+    "You work as a scam seller on Amazon selling fake airpods for full price and make {amount}.",
+    "You break into Rapunzels's kingdom and steal her hair, you sell it on the black market for {amount}."
```

Comment:
Grammar: "You break into Rapunzel's kingdom and steal her hair. You sell it on the black market for {amount}."